### PR TITLE
Improve tutorial deployment documentation

### PR DESCRIPTION
**Pull request by frankharkins**
Tries to clarify some of the steps to make it easier to follow. Related to #1390.

### CODE REVIEW

File: tutorials/README.md [hunk 1]
```diff
@@ -16,47 +16,81 @@ Check that your tutorial renders properly.
...
+   repository. Repeat this for staging and production.
+
+   1. **Get the URL**
+
+      Navigate to the tutorial in the CMS and copy the URL relative to
+      `content` (that is, the URL after
+      `https://learning-api.quantum.ibm.com/admin/content/`). The copied URL
+      should be of the form `tutorials/a00eed3f-fdba-416d-8f22-7a2522de02bc`.
+
+   2. \*\*Update `tutorials/learning-api.conf.yaml`
```
Comment (frankharkins) [inline tutorials/README.md:75]:
```suggestion
   2. **Update `tutorials/learning-api.conf.yaml`**
```

File: tutorials/README.md [hunk 2]
```diff
@@ -16,47 +16,81 @@ Check that your tutorial renders properly.
...
-tutorial and publish the lesson to take it out of draft mode.
+2. **Give our token permission to push to the tutorial**
+
+   > TODO: Improve this; I don't have permission to do this so can't give more details.
+
+   Next, we need to give the token in this repo permission to push to the new
+   tutorial. In the CMS, go to "Users", then give "Token: Qiskit/documentation"
+   permission to push to the new tutorial.
+
+3. **Link the notebook in this repo to the learning platform**
```
Comment (javabster) [inline tutorials/README.md:62]: I think this section needs to make clear you should open a followup PR to do this

File: tutorials/README.md [hunk 3]
```diff
@@ -16,47 +16,81 @@ Check that your tutorial renders properly.
...
 ## Adding new tutorials
 
 Each tutorial has its own folder in `tutorials/`. Within that folder is the content notebook
 (ending in `.ipynb`) and an optional `images` folder.
 
-To add a new tutorial to the learning platform, go to
-https://learning-api.quantum.ibm.com/admin/ and choose "Create item" (the blue
-`+` in the top-right corner). Enter all the information, but leave the content
-field blank. This will create the tutorial in "draft" mode.
+Once you've decided to add a new tutorial to the learning platform, a person
```
Comment (javabster) [inline tutorials/README.md:43]: could you make it clear in this intro that this process should first be done in staging before in prod

File: tutorials/README.md [hunk 4]
```diff
@@ -16,47 +16,81 @@ Check that your tutorial renders properly.
...
+      sub-step.
+
+4. **Push the content from this repo to the learning platform**
+
+   To push content from this repo to the learning platform, run the [Deploy
+   tutorials](https://github.com/Qiskit/documentation/actions/workflows/deploy-tutorials.yml)
+   workflow. This will push the content of **all** tutorials from the main
+   branch. Do this for both staging and production.
+
+5. **Review and publish**
```
Comment (javabster) [inline tutorials/README.md:89]: I think we should add an additional step for production - we need to remind people to set the correct access controls for network only content

**Pull request closed without merging**